### PR TITLE
[skip ci] Remove vic-ui components

### PR DESCRIPTION
**Pull request by morris-jason**
Fixes vmware/vic-product#1433

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -123,50 +123,6 @@ pipeline:
     when:
       status: success
 
-  vic-ui:
```
Comment (mdharamadas1) [inline .drone.yml]: Do we have corresponding change to include `vic-ui` artifacts in OVA build?

File: tests/manual-test-cases/Group18-VIC-UI/18-1-VIC-UI-Installer.md [hunk 2]
```diff
@@ -1,32 +0,0 @@
-Test 18-1 - VIC UI Installation
```
Comment (mdharamadas1) [inline tests/manual-test-cases/Group18-VIC-UI/18-1-VIC-UI-Installer.md]: Do we need to recreate these tests in product build? If so, we need to add an issue to do so later.

Comment (morris-jason) [reply tests/manual-test-cases/Group18-VIC-UI/18-1-VIC-UI-Installer.md]: I think this is covered in https://github.com/vmware/vic-product/issues/1701.

File: .drone.yml (hunk 1 above)
Comment (morris-jason) [reply .drone.yml]: We are no longer publishing vic-ui binaries as part of https://github.com/vmware/vic-product/pull/1742.
The changes to provision the vic-ui plugin zips archives are in https://github.com/vmware/vic-product/pull/1758.

Comment (mdharamadas1) [reply .drone.yml]: Cool, thanks!

**Pull request merged**